specification/relay/resource-manager/Microsoft.Relay/stable/2016-07-01/relay.json

```diff
@@ -403,6 +403,47 @@
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
         }
+      },
+	  "post": {
```

Comment:
[nit] formatting isn't quite right here. Please make sure you use two-space indenting for all swagger files.

Reply:
fixed.